base/JointTransform.hpp

```diff
@@ -33,13 +33,24 @@ struct JointTransformVector : public base::NamedVector<JointTransform>
      */
     void setRigidBodyStates( const base::samples::Joints& joints, std::vector<base::samples::RigidBodyState>& rbs ) const
     {
-	assert( !joints.names.empty() );
+        if (joints.names.empty()) {
+            std::stringstream ss;
+            ss << "base::JointTransformVector::setRigidBodyStates(): "
+                  "the vector 'joints.names()' is empty";
+            throw std::runtime_error(ss.str());
```

Comment:
Hm, the intention was to be able to add the current values of some variables into the exception. Will remove it.